crates/vm/backends/revm/aux.rs

```diff
@@ -0,0 +1,113 @@
+use ethrex_common::{
```

Comment:
bad name I know, but the objective was to move this inside revm

Reply:
Could be named `helpers.rs`.